### DISCUSSION TITLE
"SceneSwitcher" plugin work on FreeBSD 13?

### DISCUSSION THREAD

**Discussion by rajhlinux** [WarmUpTill/SceneSwitcher#491] [Q&A]
This is an awesome plugin. I can practically do security surveillance tasks.

Will it run on OBS FreeBSD 13?

Thanks.

**Comment by WarmUpTill** [comment]
I am not really familiar with OBS on FreeBSD 13.
I would assume that it works in theory.

As you mentioned surveillance I assume you pan on using the "Video" condition.
This condition type has a dependency to the OpenCV libraries.
If the same version of these libs are available on FreeBSD 13 as the ones used during build time it should work.
Otherwise the plugin would have to be recompiled to use whichever version of these libs is available on your system.

Hope that helps and sorry about the late reply!
